Added subscription set command

pavpavferg opened the pull request:
I could not find resources (I have two subscriptions).
Once set, everything worked

azurecla commented:
Hi __@PedroPerezMSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pepere). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@PedroPerezMSFT : Thanks for your contribution to the Azure documentation! The author, @yushwang, has been notified to review your proposed change.

yushwang commented:
Hi Pedro,

This flag is not needed, but it does trigger the update of a VPN gateway created before BGP was released to "backfill" the BGP parameters. Therefore created a misconception that this flag is needed. I will need to figure out how to explain this in the doc.

Thanks,
Yushun

PedroPerezMSFT commented:
Hey Yushun,

Good to know ☺

Thanks!
Pedro



From: Yushun Wang [mailto:notifications@github.com]
Sent: 17 June 2016 19:43
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Pedro Perez <Pedro.Perez@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Azure/azure-content] Missing EnableBGP flag when creating GW (#6845)


Hi Pedro,

This flag is not needed, but it does trigger the update of a VPN gateway created before BGP was released to "backfill" the BGP parameters. Therefore created a misconception that this flag is needed. I will need to figure out how to explain this in the doc.

Thanks,
Yushun

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-content/pull/6845#issuecomment-226849363>, or mute the thread<https://github.com/notifications/unsubscribe/AM_a-AqSxIyhpj7qkDzvxOn4_4ZcknSQks5qMuqWgaJpZM4I4U73>.

carmonmills commented:
@tysonn Please close this PR per the comments from above.

pavpavferg committed:
Added subscription set command

I could not find resources (I have two subscriptions).
Once set, everything worked

ktoliver committed:
fix typo
